packages/nimble-components/src/text-area/styles.ts

```diff
@@ -53,8 +66,11 @@ export const styles = css`
         border-radius: 0px;
         align-items: flex-end;
         border: ${borderWidth} solid transparent;
+        min-width: 100px;
+        min-height: calc(${iconSize} + ${standardPadding});
         padding: 8px;
-        transition: box-shadow ${smallDelay}, border ${smallDelay};
+        padding-right: calc(${iconSize});
```

Comment:
This is the reserved space for the icon right? Can you include a brief comment about why it's being reserved.

Reply:
Added comment